.editorconfig

```diff
@@ -8,6 +8,15 @@ charset = utf-8
 # [newline-eof]
 insert_final_newline = true
 
+# Ignore
+[*.log]
+charset = unset
+end_of_line = unset
+insert_final_newline = unset
+trim_trailing_whitespace = unset
+indent_style = unset
+indent_size = unset
+
```

Comment:
그러고보니, 이건 왜 추가하신건가요?

Reply:
```
Task :editorconfigCheck FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':editorconfigCheck'.

Could not read ...\YouAndMeApp\logs\app-2021-09-11-1.log. 
This may mean that it is a binary file and you should exclude it from editorconfig processing.
```

로그 파일 때문에 IDE의 인덴트를 맞춰주는 editorconfigCheck 가 실패하여 빌드가 되지 않습니다.
그래서 로그 파일은 적용되지 않도록 설정을 추가하였습니다.

`index_exclude_patterns = {.log}` 
이렇게 하고싶었는데, 방법을 못찾아서 [이 링크](https://newbedev.com/possible-to-ignore-exclude-file-folder-from-editorconfig)를 참고하여 작성한건데 더 좋은 방법이 있을까요?

Reply:
좋은 방법 같습니다.